### PR TITLE
add cache busting for css files

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

**Review by davidmiddletonmoj — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN npm prune --production
 FROM node:16-bullseye-slim
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
-ENV CACHE_BREAKER ${GIT_REF}
+ENV GIT_REF ${GIT_REF:-xxxxxxxxxxxxxxxxxxx}
```
Comment (petergphillips) [inline Dockerfile:24]: env variables don't need a default - see https://github.com/ministryofjustice/hmpps-template-typescript/pull/233/files
also better to use new `ENV MY_VAR=my_value` syntax - see _Alternative syntax_ section in https://docs.docker.com/engine/reference/builder/#env

File: backend/config.ts [hunk 2]
```diff
@@ -38,6 +38,7 @@ export const app = {
     enabledPrisons: process.env.HOMEPAGE_REDIRECT_ENABLED_PRISONS || '',
     scheduleRedirectForPrisons: process.env.HOMEPAGE_SCHEDULE_REDIRECT_FOR_PRISONS || '',
   },
+  gitRef: process.env.GIT_REF || '',
```
Comment (petergphillips) [inline backend/config.ts:41]: should really be `requiredInProduction` - see https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/config.ts#L35

**Review by petergphillips — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (whitfield-mj) [reply Dockerfile:24]: Ha, I didn't have one but included it because it was in the template!

File: backend/config.ts (hunk 2 above)
Comment (whitfield-mj) [reply backend/config.ts:41]: Yes, DPS does not have a lot of things that the template does

**Pull request merged**